lua/cmp/view/docs_view.lua

```diff
@@ -16,6 +16,7 @@ docs_view.new = function()
   self.window:option('linebreak', true)
   self.window:option('scrolloff', 0)
   self.window:option('wrap', true)
+  self.window:buffer_option("filetype", "cmp-docs-view")
```

Comment:
Could you change it as `cmp_documentation`?

Reply:
Changes are complete